Name: 31globalization.js
Version: 2.1.0-1
Author: The Resetter
Description: Add "India" to Regional Settings -> Phone Numbers

--- .orig/usr/palm/frameworks/globalization/submission/31globalization.js
+++ /usr/palm/frameworks/globalization/submission/31globalization.js
@@ -131,7 +131,7 @@
 if(!nextProp)return prev+identify(currentNode)||prev||""}return prev||""}while(template.length){nextKey=template.match(pattern);if(nextKey){parts.push(template.slice(0,nextKey.index));parts.push(nextValue(nextKey[1],nextKey[2],nextKey[3]));template=template.slice(nextKey.index+nextKey[0].length)}else{parts.push(template);template=""}}return parts.join("")}};Globalization.Format.Template.Pattern=/(.?)(#\{(.*?)\})/;Globalization.Format.getPhoneFormatExamples=function(formatsRegion){var region,regionSettings,styles,entry;formatsRegion=formatsRegion||Globalization.Locale.getCurrentPhoneRegion();region=Globalization.Phone._getNormalizedPhoneRegion(formatsRegion);regionSettings=Globalization.Phone._loadFormatsFile(region);styles=[];if(regionSettings.phoneNumberFormats.styles)for(style in regionSettings.phoneNumberFormats.styles)if(regionSettings.phoneNumberFormats.styles[style].example)styles.push({key:style,value:regionSettings.phoneNumberFormats.styles[style].example});
 Utils.releaseAllJsonFiles(Globalization.Locale.phoneRegion);return styles};
 Globalization.Format.getPhoneFormatRegions=function(){return[{countryCode:"us",countryName:"US, Canada, Caribbean"},{countryCode:"it",countryName:"Italy"},{countryCode:"fr",countryName:"France"},{countryCode:"gb",countryName:"United Kingdom"},{countryCode:"ie",countryName:"Ireland"},{countryCode:"de",countryName:"Germany"},{countryCode:"nl",countryName:"Netherlands"},{countryCode:"be",countryName:"Belgium"},{countryCode:"lu",countryName:"Luxembourg"},{countryCode:"es",countryName:"Spain"},{countryCode:"mx",
-countryName:"Mexico"},{countryCode:"cn",countryName:"China"},{countryCode:"au",countryName:"Australia"},{countryCode:"sg",countryName:"Singapore"},{countryCode:"nz",countryName:"New Zealand"}]};
+countryName:"Mexico"},{countryCode:"cn",countryName:"China"},{countryCode:"au",countryName:"Australia"},{countryCode:"sg",countryName:"Singapore"},{countryCode:"nz",countryName:"New Zealand"},{countryCode:"in",countryName:"India"}]};
 Globalization.Phone._substituteDigits=function(part,formats,mustUseAll){var format;var formatted="";var partIndex=0;if(!part)return formatted;if(typeof formats==="object"){if(part.length>formats.length)throw"part "+part+" is too big. We do not have a format template to format it.";format=formats[part.length-1]}else format=formats;for(var i=0;i<format.length;i++)if(format.charAt(i)=="X"){formatted+=part.charAt(partIndex);partIndex++}else formatted+=format.charAt(i);if(mustUseAll&&partIndex<part.length-
 1)throw"too many digits in "+part+" for format "+format;return formatted};
 Globalization.Format._formatPhoneNumberWithParams=function(phoneNumberModel,params){var region,formatsRegion,regionSettings,format,temp,templates,fieldName,countryCode,isWhole,style,field,formatted="",styleNumber,iter,i;try{isWhole=true;if(params){if(params.mcc){countryCode=Globalization.Phone.getMobileCountryCode(params.mcc);formatsRegion=PhoneData.mccToCountry[params.mcc]}else if(params.locale){parsedLocale=Globalization.Locale.parseLocaleString(params.locale);formatsRegion=parsedLocale.region}if(params.partial&&
